Examples/UIExplorer/UIExplorerApp.ios.js

```diff
@@ -85,12 +88,20 @@ class UIExplorerApp extends React.Component {
   _renderOverlay: Function;
   _renderScene: Function;
   _renderCard: Function;
+  constructor(props) {
+    super(props);
+    this._handleOpenExample = this._handleOpenExample.bind(this);
+    this.state = {openExampleName: props.openExampleName};
+  }
   componentWillMount() {
     this._renderNavigation = this._renderNavigation.bind(this);
     this._renderOverlay = this._renderOverlay.bind(this);
     this._renderScene = this._renderScene.bind(this);
     this._renderCard = this._renderCard.bind(this);
   }
+  componentDidMount() {
+    setTimeout(this._handleOpenExample, 500);
```

Comment:
Same notes as the `.android` file.

Can we also rename the `openExampleName` prop to something that will make people know why we did it this way? Usually people can simply use the built-in linking code. We only need to do it this way to support appetize